dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/WorkFlowLineageMapper.xml

```diff
@@ -163,6 +163,25 @@
         AND a.task_type = 'DEPENDENT'
     </select>
 
+    <select id="queryDependentOnlineProcessDefinitionByProcessDefinitionCode" resultType="DependentProcessDefinition">
```

Comment:
Due to the exists mapper `queryDependentProcessDefinitionByProcessDefinitionCode ` only use in one place, can you please replace it instead of add a new entrepoint named `queryDependentOnlineProcessDefinitionByProcessDefinitionCode `

Reply:
Create complement dependent command use the method 'queryDependentOnlineProcessDefinitionByProcessDefinitionCode'  only，modify the sql。